_data/params.csv

```diff
@@ -8,7 +8,7 @@ wbraid, Google AdWords,https://support.google.com/google-ads/answer/10417364,
 gad_source, Google AdWords,https://support.google.com/google-ads/answer/13327296,
 srsltid, Google Analytics,https://support.google.com/analytics/answer/11479699,
 twclid, Twitter,https://business.twitter.com/en/help/campaign-measurement-and-analytics/conversion-tracking-for-websites.html,
-yclid, Yahoo,https://ads-help.yahoo-net.jp/s/article/H000044650,
+yclid, Yandex,https://yandex.com/support/metrica/data/yclid.html,
```

Comment:
`yclid` is actually listed in the Yahoo! documentation as well.

https://ads-help.yahoo-net.jp/s/article/H000044650?language=ja

Maybe we can use a slash to indicate the fact that it is used by multiple platforms `Yahoo / Yandex`

Reply:
Then si (which is used by both YouTube (when sharing/copying the link) and Spotify) would need the same treatment.